### ISSUE TITLE
Cannot change scaling or font size after image installation

### ISSUE THREAD

**Issue by nintyfan** [CypherpunkArmory/UserLAnd#806]
I install Debian with XFCE, but text is too big. Other think is I don't have virtual keyboard access, so cannot modify any files.

**Comment by bubnikv** [comment]
> I install Debian with XFCE, but text is too big. Other think is I don't have virtual keyboard access, so cannot modify any files.

I would be happy if the UserLAnd application assisted me to configure the default XWindow font size and possibly the VNC server virtual screen resolution as well.

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
